amundsen_application/static/js/components/Feedback/FeedbackForm/BugReportFeedbackForm/index.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
```

Comment:
I'm beginning to think there are places where we probably don't need a new folder for each subcomponent. What do you think about a structure like:

```
components/Feedback/AbstractFeedbackForm.tsx
components/Feedback/BugReportFeedbackForm.tsx
components/Feedback/FeatureRequestFeedbackForm.tsx
components/Feedback/RatingFeedbackForm.tsx
components/Feedback/FeatureRequestFeedbackForm.tsx
components/Feedback/Feedback.tsx
```

Reply:
We might not need it, but I'd rather be consistent. In addition each component is eventually going to have a `tests` folder nested inside of their directories.